README.rst

```diff
@@ -48,20 +48,32 @@ Adjust according to your operating system or personal preferences.
 #. Install a virtual environment for development (Python 2 or Python 3, you choose!)::
 
     $ sudo pip install virtualenvwrapper
+
+#. Follow the instructions from http://virtualenvwrapper.readthedocs.org/en/latest/install.html#basic-installation.
+   You will need to source the virtualenvwrapper.sh file for the following commands to work::
+
     $ mkvirtualenv --python=python3 kolibri
     $ workon kolibri
 
+#. Install the other packages needed for development::
+
+   $ sudo pip install -r requirements/dev.txt
```

Comment:
Not sure if it has an effect, did you render it? Indentations of source code are normally 4 spaces.